src/api/apiService.js

```diff
@@ -78,7 +78,8 @@ class Api {
     const requestData = {
       method: 'get',
       headers: {
-        Accept: 'application/json'
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
```

Comment:
@aguscha333 @bernabe9 should we be adding this to the base?

Reply:
Do you mean this? https://github.com/rootstrap/react-redux-base/commit/704d59e1e947a91fd4358e60cce4f2655d4c942c

Reply:
I cloned it before that fix, thanks @aguscha333 !